### PR TITLE
Create a temporary job on plumbing 😏

### PR DESCRIPTION
**Pull request by vdemeester**
# Changes

The idea is to be able to test if the `:mario` image fails.

/cc @afrittoli 

Signed-off-by: Vincent Demeester <vdemeest@redhat.com>

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [x] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/pipeline/blob/master/CONTRIBUTING.md)
for more details._

### CODE REVIEW

File: prow/config.yaml [hunk 1]
```diff
@@ -63,6 +63,40 @@ tide:
 
 presubmits:
   tektoncd/plumbing:
+  - name: test-dogfooding-latest
+    agent: kubernetes
+    always_run: true
+    rerun_command: "/test pull-tekton-cli-build-tests"
```
Comment (dibyom) [inline prow/config.yaml:69]: Is this intentional?

File: prow/config.yaml [hunk 2]
```diff
@@ -63,6 +63,40 @@ tide:
...
+    trigger: "(?m)^/test (all|pull-tekton-cli-build-tests),?(\\s+|$)"
```
Comment (dibyom) [inline prow/config.yaml:70]: same as above...cli seems unintentional

**Comment by vdemeester** [comment]
/hold

Yeah.. I need to fix this

File: prow/config.yaml [hunk 3]
```diff
@@ -63,6 +63,41 @@ tide:
 
 presubmits:
   tektoncd/plumbing:
+  - name: pull-dogfooding-mario-test
+    agent: kubernetes
+    always_run: true
```
Comment (afrittoli) [inline prow/config.yaml:68]: I think we should leave this as an "on-demand" only job - i.e. do not run unless someone does "/test pull-dogfooding-mario-test"

File: prow/config.yaml [hunk 4]
```diff
@@ -63,6 +63,41 @@ tide:
...
+    decorate: true
+    rerun_command: "/test pull-dogfooding-mario-test"
+    trigger: "(?m)^/test (all|pull-dogfooding-mario-test),?(\\s+|$)"
```
Comment (afrittoli) [inline prow/config.yaml:71]: same here, I would not include "all" here - otherwise this will run on all plumbing patches

File: prow/config.yaml [hunk 5]
```diff
@@ -63,6 +63,41 @@ tide:
...
+        - "--clean"
+        - "--job=$(JOB_NAME)"
+        - "--repo=github.com/$(REPO_OWNER)/$(REPO_NAME)=$(PULL_REFS)"
+        - "--root=/go/src"
+        - "--service-account=/etc/test-account/service-account.json"
+        - "--upload=gs://tekton-prow/pr-logs"
+        - "--" # end bootstrap args, scenario args below
+        - "--" # end kubernetes_execute_bazel flags (consider following flags as text)
+        - "echo"
+        - "this works, or... does it ?"
```
Comment (afrittoli) [inline prow/config.yaml:87]: LoL

File: prow/config.yaml (hunk 3 above)
Comment (vdemeester) [reply prow/config.yaml:68]: make sense

**Pull request merged**